spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -20,7 +20,7 @@ import java.util.Date
 
 import org.apache.commons.configuration2.Configuration
 import org.apache.spark.SPARK_VERSION
-import org.apache.spark.sql.functions.lit
+import org.apache.spark.sql.functions._
```

Comment:
There are these imports at the top that don't seem to be used. Can we remove them?
```
import java.text.SimpleDateFormat
import java.util.Date
```

Reply:
Now they are used again 😄